### ISSUE TITLE
F-Droid inclusion

### ISSUE THREAD

**Issue by shuvashish76** [mardous/WhatSave#5]
Hello, would you consider publishing the app on F-Droid please?

**Comment by mardous** [comment]
Hello. Yes, **it is part of my plans**, but lately due to time issues it has become complicated. However, I hope to be able to complete it in the next few days.

**Comment by shuvashish76** [comment]
ThankYou. Documentation for the inclusion : https://gitlab.com/fdroid/fdroiddata/blob/master/CONTRIBUTING.md

**Comment by mardous** [comment]
Thanks

**Comment by mardous** [comment]
The app has been added to F-Droid :), although it will take a few days for it to be visible.

**Comment by shuvashish76** [comment]
https://gitlab.com/fdroid/wiki/-/wikis/FAQ#how-long-does-it-take-for-my-app-to-show-up-on-website-and-client

And feel free to use their badge: https://gitlab.com/fdroid/artwork/-/blob/master/badge/README.md
or if you prefer shield badge : https://shields.io/badges/f-droid-version

**Comment by BabyOilJohnson** [comment]
Hi,
@mardous what's the difference b/w normal and fdroid?

**Comment by mardous** [comment]
> https://gitlab.com/fdroid/wiki/-/wikis/FAQ#how-long-does-it-take-for-my-app-to-show-up-on-website-and-client
> 
> And feel free to use their badge: https://gitlab.com/fdroid/artwork/-/blob/master/badge/README.md or if you prefer shield badge : https://shields.io/badges/f-droid-version

Basically, the APK marked as "fdroid" has been compiled by F-Droid and is the one available for download from that site. It is smaller in size because it does not contain any Firebase dependencies and the reason it is included here is so that the internal app update system can work on the F-Droid version.

The "normal" one is the preferred APK file for any non-F-Droid download.

**Comment by mardous** [comment]
> Hi, @mardous what's the difference b/w normal and fdroid?

Basically, the APK marked as "fdroid" has been compiled by F-Droid and is the one available for download from that site. It is smaller in size because it does not contain any Firebase dependencies and the reason it is included here is so that the internal app update system can work on the F-Droid version.

The "normal" one is the preferred APK file for any non-F-Droid download.

**Comment by BabyOilJohnson** [comment]
I know this dumb question
So that means "fdroid" ver is not signed by dev(you) right?

**Comment by mardous** [comment]
> I know this is dumb question. So that means "fdroid" ver is not signed by dev(you) right?

That's right, the "fdroid" version has been signed by F-Droid. Then manually downloaded and placed here so that the app's update system can detect the APK.
